core/CronArchive/QueueConsumer.php

```diff
@@ -397,18 +397,25 @@ private function shouldSkipArchiveBecauseLowerPeriodOrSegmentIsInProgress(array
                 continue;
             }
 
+            if (empty($archiveBeingProcessed['idSite'])
```

Comment:
Haven't had a deeper look at the code, but should it actually happen that `idSite` isn't set for an archiving process? If so wouldn't we need to check if `idSite` of both aren't empty here?  🤔

Reply:
It's a bit confusing, but here `$archiveBeingProcessed` are the parameters for a climulti:request, while `$archiveToProcess` is a row in archive_invalidations. So `idSite` might not be set in a `climulti:request` execution, but `idsite` will always be in `$archiveToProcess`.